internal/grpcutil/util.go

```diff
@@ -16,18 +16,17 @@
  *
  */
 
-package binarylog
+package grpcutil
 
 import (
 	"errors"
 	"strings"
 )
 
-// parseMethodName splits service and method from the input. It expects format
+// ParseMethodName splits service and method from the input. It expects format
```

Comment:
Couple of nits here:
The grpcutil package now has a file named `target.go` which exports a function `ParseTarget`. So, maybe to keep things consistent, we should name this file as `method.go` and the function `ParseMethod`.